docs/examples/kube2iam.rst

```diff
@@ -0,0 +1,242 @@
+
+.. code-block:: bash
+
+    helm version
+    Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
+    Server: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
+
+Setup
+~~~~~
+
+Create instance IAM policy
+++++++++++++++++++++++++++
+
+Every instance that will run the kube2iam pod needs to have an specific
+IAM policy attached to the IAM instance role of that instance.
+
+The following Terraform project creates an IAM policy that will give 
+instances the ability to assume roles. The assume role is restricted to
+only have access to roles deployed in a specific path. By doing this, we can
+limit the amount of roles an instance can assume to only the roles that it really
+needs to.
+The Terraform project has 2 inputs ``aws_region`` and ``cluster_name``.
+The projects also has 2 outputs defined the ``ARN`` and ``path`` of the IAM policy.
+The ARN is what you need to give to Tarmak and the path is needed to be
+able to deploy your roles for the pods in the correct path.
+
+.. code-block:: none
+
+    terraform {}
+
+    provider "aws" {
+        region = "${var.aws_region}"
+    }
+
+    variable "aws_region" {
+        description = "AWS Region you want to deploy it in"
+        default     = "eu-west-1"
+    }
+
+    variable "cluster_name" {
+        description = "Name of the cluster"
+    }
+
+    data "aws_caller_identity" "current" {}
+
+    resource "aws_iam_policy" "kube2iam" {
+        name        = "kube2iam_assumeRole_policy_${var.cluster_name}"
+        path        = "/"
+        description = "Kube2IAM role policy for ${var.cluster_name}"
+
+        policy = "${data.aws_iam_policy_document.kube2iam.json}"
+    }
+
+    data "aws_iam_policy_document" "kube2iam" {
+        statement {
+            sid = "1"
+
+            actions = [
+            "sts:AssumeRole",
+            ]
+
+            effect = "Allow"
+
+            resources = [
+            "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/kube2iam_${var.cluster_name}/*",
+            ]
+        }
+    }
+
+    output "kube2iam_arn" {
+        value = "${aws_iam_policy.kube2iam.arn}"
+    }
+
+    output "kube2iam_path" {
+        value = "${aws_iam_policy.kube2iam.path}"
+    }
+
+
+You can run the Terraform project the following way:
+
+.. code-block:: bash
+
+    terraform init
+    terraform apply -var cluster_name=example -var region=eu-west-1
+
+Attach instance policy
+++++++++++++++++++++++
+
+Add the created IAM policy ARN to your tarmak config. You can do this by
+adding `additional IAM policies <https://docs.tarmak.io/user-guide.html#additional-iam-policies>`_.
+
+Deploy kube2iam
++++++++++++++++
+
+With `HELM <https://www.helm.sh/>`_ it is easy to deploy kube2iam with 
+the correct settings.
+
+You can deploy it with the following command:
+
+.. code-block:: bash
+
+    helm upgrade kube2iam stable/kube2iam \
```

Comment:
This chart doesn't exist yet?

Reply:
that chart exists https://github.com/kubernetes/charts/tree/master/stable/kube2iam